src/components/Modals/FiatRamps/config.ts

```diff
@@ -23,19 +27,20 @@ export interface SupportedFiatRampConfig {
   getBuyAndSellList: () => Promise<[FiatRampAsset[], FiatRampAsset[]]>
   onSubmit: (action: FiatRampAction, asset: string, address: string) => void
   minimumSellThreshold?: number
+  supportsBuy: boolean
+  supportsSell: boolean
```

Comment:
We should add a new `minimumBuyThreshold` property, see:
https://help.coinbase.com/en/coinbase/trading-and-funding/buying-selling-or-converting-crypto/what-is-the-minimum-amount-of-cryptocurrency-that-i-can-purchase

Since we don't know which currency a user might have in their account, we can use GBP as a reference and err on the side of caution by setting it to 3 (2 GBP = 2.4 USD currently).

When multi currency is out, we might want to think about making this dynamic depending on the user's currency (2GBP or 2EUR to USD), but that'd be a product decision and isn't going to consistently work, as some users might have USD set up in the app but use EUR in coinbase)

Reply:
At first I agreed with this. As I was working to put it in, I realized that we have no context to how much fiat they will use to buy and that is a purely Coinbase/gem/banxa issue. We can't know how much fiat they have in their coinbase account to buy, or how much they will use on a credit card to purchase crypto.

So, while selling, we can know how much crypto they have to sell and if it is below the minimum threshold, we can warn them. When buying the same dynamic doesn't exist because we do not handle fiat nor know anything about the user's fiat balances.